include/openssl/pkcs7.h

```diff
@@ -498,6 +498,11 @@ PKCS7_get_recipient_info(PKCS7 *p7);
 OPENSSL_EXPORT OPENSSL_DEPRECATED PKCS7_RECIP_INFO *PKCS7_add_recipient(
     PKCS7 *p7, X509 *x509);
 
+// PKCS7_get0_signers() retrieves the signer's certificates from p7, it does not
```

Comment:
```suggestion
// PKCS7_get0_signers retrieves the signer's certificates from p7, it does not
```

nitpick: doc comments don't use parens